supportedBackends/htcondor/src/main/scala/cromwell/backend/impl/htcondor/HtCondorInitializationActor.scala

```diff
@@ -0,0 +1,65 @@
+package cromwell.backend.impl.htcondor
+
+import cromwell.backend.BackendLifecycleActor.WorkflowAbortResponse
+import cromwell.backend.validation.RuntimeAttributesKeys._
+import cromwell.backend.validation.Validation._
+import cromwell.backend.{BackendConfigurationDescriptor, BackendWorkflowDescriptor, BackendWorkflowInitializationActor}
+import wdl4s.util.TryUtil
+import wdl4s.{Call, ThrowableWithErrors, WdlExpression}
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+import scalaz.Scalaz._
+
+class HtCondorInitializationActor(override val workflowDescriptor: BackendWorkflowDescriptor,
+                                  override val calls: Seq[Call],
+                                  override val configurationDescriptor: BackendConfigurationDescriptor) extends BackendWorkflowInitializationActor {
+  /**
+    * Abort all initializations.
+    */
+  override def abortInitialization: Future[WorkflowAbortResponse] = ???
+
+  /**
+    * Validate that this WorkflowBackendActor can run all of the calls that it's been assigned
+    */
+  override def validate: Future[Unit] = {
+    //TODO: Expressions need to be resolved. EvalFunctions is needed. TBD in the PR.
```

Comment:
I need some opinions on how to resolve the evaluation of run-time attributes expressions.